### ISSUE TITLE
Dealing with linkouts and cross-references in the Monarch App

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/monarch-app#521]
There are three kinds of "links" between related resources: Mappings, cross-references and linkouts:

<img width="1242" alt="image" src="https://github.com/monarch-initiative/monarch-app/assets/7070631/dfc27bbf-d564-4649-8692-d80c04b7b2eb">

Linkouts are not just "links to related content", but a very important component of open science that represent "a global knowledge graph of information" and, perhaps even more importantly, a "global network of partners and collaborators to tackle the problems of our times". It is, in my view, therefore worth spending a few minutes thinking about how to present these "relationships" in a way that most ideally supports both information discovery across partner sites and global collaborative efforts. 

TLDR: Rather than just "adding linkouts" to external resources, I think we need a proper system.

There are at least three problems:

- The representation of linkouts (as opposed to xrefs) in the ontology (I need to discuss this with Trish and Sabrina)
- The attachment of metadata to the xrefs/linkouts (this can probably wait, but is important in the long run because we need to actively acknowledge our collaborators efforts more) - right now the APIs cant do that very well, and we absolutely need this (both for synonyms and xrefs)
- The presentation of linkouts on the site. Here I would prefer if the UI was source aware, like say https://obofoundry.org/ontology/ro.html (where, in the case of a linkout or perhaps even xref, you have a logo of the external resource and click on it to obtain the external content).

This is not purely technical - this is about showing that we are a really good collaborating partner, acknowledging everywhere the efforts of our partners (which are huge) - so it is, in some sense, political as well as technical.

To focus this important discussion, I would like to suggest the following two action items for this ticket:

1. Create a list of all "partner" (whether officially or in mission/spirit) we would like to "link-out" to.
2. Decide how to best present those links on our website

I will make some parallel tickets that talk about the representation of the information in the API.

Related discussion on Reactome linkouts #465 

#### Diseases

1. ClinGen
2. MedGen
3. NORD
4. GARD
5. Open Targets
6. OLS
7. Orphanet
8. OMIM
9. Disease Ontology
10. NCIT

#### Pathways

1. Reactome

#### Phenotypes

1. ZFIN
2. MGI
3. HPO website
4. FlyBase

### Current suggestions for display

1. @matentzn: I like to have themed buttons, ideally with logos, that are visible on all pages with a linkout, e.g. https://obofoundry.org/ontology/ro.html: 
<img width="378" alt="image" src="https://github.com/monarch-initiative/monarch-app/assets/7070631/079ce774-a5a4-45ac-8459-dbe457cdc9c6">. When hovering over the button, basic provenance about the metadata should be displayed.

### Related issues

- https://github.com/monarch-initiative/mondo/issues/7071
